duo_client/admin.py

```diff
@@ -2560,6 +2561,7 @@ def update_admin(self, admin_id,
         phone - <str:phone number> (optional)
         password - Deprecated; ignored if specified.
         password_change_required - <bool|None:Whether admin is required to change their password at next login> (optional)
+        status - the status of the adminstrator
```

Comment:
I would agree that the field should be optional.    However, i believe that according to the API documentation the values can be
"Active", "Disabled","Expired","Pending Activation"

Reply:
yes, those are the possible statuses, but you can't put an admin in the latter two directly I believe.

Reply:
It also mentions that you cannot change an administrator with "Owner" role to "Disabled" through the api.  So - any idea on how to modify an administrator that has the "Owner" role and has not accessed the system in a specified time (line 60 days)? through an automation (using the api) ?

Reply:
This is not possible through the API. Duo wants to prevent a malicious actor who has gotten API credentials from locking Owners out of a Duo account.  Deactivation of Owners has to go through our Support organization, so they can verify the legitimacy of the request.